test/unit/state/frameStateUtilTest.js

```diff
@@ -32,87 +42,7 @@ describe('frameStateUtil', function () {
 
   after(function () {
     mockery.disable()
```

Comment:
this file has a LOT of changes- mostly organizing the order of the methods being tested to actually match the order in frameStateUtil.js